### ISSUE TITLE
[PROB] EGOI 2023 Problems

### ISSUE THREAD

**Issue by snowysecrett** [ojuz/ojuz-contribute#13]
EGOI 2023.
Statements: https://egoi23.se/competition/tasks

All other files: https://github.com/Kodsport/egoi23-testdata/tree/main

**Comment by tncks0121** [comment]
It seems I have to rewrite many checkers and interactive graders to make it work on our website. Given that Kattis provides grading, I think uploading this will be delayed

**Comment by dbsic211** [comment]
Makes sense to me. In particular, this may be true for day 2 problem 4 -
Guessing game. But for the other problems, hopefully it is not too big of
an issue - just standard I/O.

**Comment by tncks0121** [comment]
https://oj.uz/problems/source/egoi2023
